sdk/keyvault/Azure.Security.KeyVault.Certificates/src/KeyVaultCertificateIdentifier.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Security.KeyVault.Certificates
+{
+    /// <summary>
+    /// Information about a <see cref="KeyVaultCertificate"/> parsed from a <see cref="Uri"/>.
+    /// You can use this information when calling methods of a <see cref="CertificateClient"/>.
+    /// </summary>
+    public readonly struct KeyVaultCertificateIdentifier
```

Comment:
It seems like these `KeyVault*Identifier` structs are all identical to `KeyVaultIdentifier` with the exception of name. Did we consider having each library use `KeyVaultIdentifier` instead of creating these - especially considering we do no validation that the provided id is of the correct resource type?

Reply:
No validation of the resource type is the desired behavior for now. That's client-side validation, and we should let the service validate as needed.

The overhead is negligible (especially given these are structs, so we're just shuffling references around anyway), so I'd rather keep the code shared as it has been since the original 4.0 previews.